gogio/androidbuild.go

```diff
@@ -468,6 +471,19 @@ func exeAndroid(tmpDir string, tools *androidTools, bi *buildInfo, extraJars, pe
 				<category android:name="android.intent.category.LAUNCHER" />
 			</intent-filter>
 		</activity>
+		{{if .HasService}}
+               <service android:name="org.gioui.GioForegroundService"
```

Comment:
We can't submit this change before your change that adds `GioForegroundService`, right?

Reply:
correct